src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -10,25 +14,77 @@ Not all spatial alignment strategies support all platforms. See the chart below
 | QR Code Detection      | x          |            | x       | x   |
 | ArUco Marker Detection |            | x          | x       | x   |
 
-## Application Flow
+## Key Concepts
+
+Before diving into the abstractions, we operate on two concepts when speaking of localization:
+
+- **Coordinate Space:** When a rotation/position is meant to be relative to a specific coordinate (location in the real world), we say it is in coordiante space. These roations/positions can be sent across the network and understood by each device that understand what coordiante they are associated with.
+- **Application World Space:** The rotations/positions that are set in Unity to `.position` and `.rotation` properties of `Transform` objets, are specific to the applications own "world space". This "world space" is used by the application to determine how to lay out content relative to each other; and this "world space" itself is relative to the positon/rotation of device when the application was launched.
```

Comment:
subjective suggestion: it seems worth calling out that this coordinate system is specific to the single device. I also think that "world space" is a buzzword that may be well understood by the AR community, but may not be well understood by a new AR developer. It may be worth changing Application World Space to Local Application Space.

Reply:
added local